app/src/main/java/com/futurice/freesound/common/utils/Parcels.kt

```diff
@@ -0,0 +1,92 @@
+
+package com.futurice.freesound.common.utils
+
+import android.os.Parcel
+import android.os.Parcelable
+import java.math.BigDecimal
+import java.math.BigInteger
+import java.util.*
+import kotlin.collections.ArrayList
+
+interface KParcelable : Parcelable {
```

Comment:
Could you please add a sprinkling of comments here (it doesn't have to be full Javadoc), so it's clearer what's going on.